src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -342,12 +342,15 @@ ComputeTimeDerivative<Metavariables>::apply(
   static constexpr size_t volume_dim = Metavariables::volume_dim;
   using system = typename Metavariables::system;
   using variables_tag = typename system::variables_tag;
+  using dt_variables_tag = db::add_tag_prefix<::Tags::dt, variables_tag>;
   using partial_derivative_tags = typename system::gradient_variables;
   using flux_variables = typename system::flux_variables;
   using compute_volume_time_derivative_terms =
       typename system::compute_volume_time_derivative_terms;
 
   const Mesh<volume_dim>& mesh = db::get<::domain::Tags::Mesh<volume_dim>>(box);
+  const ::dg::Formulation& dg_formulation =
```

Comment:
- should not need to make a reference for an enum
- check for other places below that grab the enum from the Databox, that should now just use your new variable (e.g. line 375)